performance/SwiftUIPerformanceQuickstart/PerformanceExample/Shared/Process.swift

```diff
@@ -0,0 +1,217 @@
+//
+//  Copyright 2021 Google LLC
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import FirebasePerformance
+import Vision
+
+class Process: ObservableObject {
```

Comment:
Maybe name this `LongRunningProcess`? Also, do you think it'd be a good idea to separate the class responsible for uploading the image from the class responsible for classifying it?

Reply:
I didn't implement this because I felt the classification task was really just one "relatively simple" function, to which the `precision` user setting is passed. The `categories` is more associated with the image than with a `Classifier`. Additionally, this would require message passing between `Classifier` and `LongRunningProcess` to update the status of the ideally "sole running task". Doing this might make the app more complicated than it needs to be? I would appreciate hearing your thoughts on this, especially if you were thinking of some advantages to separating them.

Reply:
SGTM